### ISSUE TITLE
Problem with addRequiredFields adding id and __typename to edges, pageInfo and parent of edges

### ISSUE THREAD

**Issue by EloB** [monojack/graphql-normalizr#10]
I'm not sure this is intended behaviour?

See here:
https://codesandbox.io/s/nw5xqr0vkj

The graphql api that I use doesn't pass the validation because of that and won't return any output.

**Comment by EloB** [comment]
Maybe I'm doing something wrong. Here is my simple fetcher.

```
import { ajax } from 'rxjs/ajax';

export const graphql = ({
  url,
  query,
  variables,
  headers,
}) => ajax.post(
  url,
  {
    query,
    variables,
  },
  {
    ...headers,
    'Content-Type': 'application/json',
  },
);
```

**Comment by EloB** [comment]
```gql
{
  items {
    __typename
    id
    edges {
      __typename
      id
      node {
        __typename
        id
        name
      }
    }
  }
}
```

**Comment by EloB** [comment]
Related https://github.com/monojack/graphql-normalizr/pull/13
